xrmocap/model/mvp/projattn.py

```diff
@@ -0,0 +1,255 @@
+# yapf: disable
+
+import Deformable as DF
+import math
+import torch
+import torch.nn.functional as F
+import warnings
+from torch import nn
+from torch.autograd import Function
+from torch.autograd.function import once_differentiable
+from torch.nn.init import constant_, xavier_uniform_
+
+# yapf: enable
+
+
+class ProjAttn(nn.Module):
+
+    def __init__(self,
+                 d_model=256,
+                 n_levels=4,
+                 n_heads=8,
+                 n_points=4,
+                 projattn_pos_embed_mode='use_rayconv'):
+        """Projective Attention Module Modified from DeformableConvV2 and
+        Deformable DETR https://github.com/chengdazhi/Deformable-
+        Convolution-V2-PyTorch/tree/pytorch_1.0.0
+        https://github.com/fundamentalvision/Deformable-DETR.
+
+        Args:
+            d_model (int, optional):
+                hidden dimension. Defaults to 256.
```

Comment:
Capitalize the first letter?

Reply:
Fixed in [ad43f3b](https://github.com/openxrlab/xrmocap/pull/21/commits/ad43f3b61eb55fb3c7998d07ae3e7b87c871347f).